x/swaprouter/types/keys.go

```diff
@@ -11,4 +18,35 @@ const (
 var (
 	// KeyNextGlobalPoolId defines key to store the next Pool ID to be used.
 	KeyNextGlobalPoolId = []byte{0x01}
+
+	// SwapModuleRouterPrefix defines prefix to store pool id to swap module mappings.
+	SwapModuleRouterPrefix = []byte{0x02}
 )
+
+// ModuleRouteToBytes serializes moduleRoute to bytes.
+func ModuleRouteToBytes(moduleRoute ModuleRoute) []byte {
+	bz, err := proto.Marshal(&moduleRoute)
+	if err != nil {
+		panic(err)
+	}
+	return bz
```

Comment:
Is this currently used somewhere? If not, where / how is it planning to get used?

Reply:
Nice catch, this is something I was trying at the start but ended up moving away from. Removed